Exec/RegTests/Soot-Flame/soot-flame.inp

```diff
@@ -61,10 +60,11 @@ amr.plot_file       = plt # root name of plotfile
 # SOOT MODELING
 soot.incept_pah = A2 # Soot inception species
 soot.v = 0
-soot.max_dt_rate = 0.01
 soot.num_subcycles = 1
 soot.temp_cutoff = 350.
 soot.conserve_mass = false
+# UNUSED PARMPARSE
+soot.max_dt_rate = 0.01
```

Comment:
@ldowen why are these not used? deprecated or renamed or something else?

Reply:
Deprecated